kong/templates/nginx_kong.lua

```diff
@@ -29,11 +29,13 @@ lua_code_cache ${{LUA_CODE_CACHE}};
 lua_socket_pool_size ${{LUA_SOCKET_POOL_SIZE}};
 lua_max_running_timers 4096;
 lua_max_pending_timers 16384;
-lua_shared_dict kong 4m;
-lua_shared_dict cache ${{MEM_CACHE_SIZE}};
-lua_shared_dict process_events 1m;
-lua_shared_dict cluster_events 1m;
-lua_shared_dict cassandra 5m;
+lua_shared_dict kong                4m;
```

Comment:
All the locks should be in the `cache_locks`, and if they are not, that would be a mistake